bodhi/client/bindings.py

```diff
@@ -273,7 +273,7 @@ def comment(self, update, comment, karma=0, email=None):
                   'csrf_token': self.csrf()})
 
     @errorhandled
-    def save_override(self, nvr, duration, notes):
+    def save_override(self, **kwargs):
```

Comment:
This change is not backwards compatible, and the bindings aren't only used by the Bodhi client. I think instead we should add a new ```edited``` parameter that defaults to ```False``` to preserve backwards compatibility, while continuing to allow ```nvr```, ```duration```, and ```notes``` to be positional arguments.